### ISSUE TITLE
Aviationstack (Independent Publisher) issue

### ISSUE THREAD

**Issue by elroyairmed** [microsoft/PowerPlatformConnectors#2874]
Api Name - shared_aviationstackip

Bug description -
I get this error message when trying to test the powerapp connector:
"Your current subscription plan does not support this API function."

**Comment by troystaylor** [comment]
Hi, can you tell me which plan you have subscribed to? When I created the connector, the Free plan offered the API, but maybe it doesn't anymore.

**Comment by elroyairmed** [comment]
Yes it is the free plan. I wanted to at least test the functionality before committing to a subscription. Cheers, Mark

**Comment by elroyairmed** [comment]
I've paid for the subscription to test it out. Are the dates/times reported in ISO 8601 format as per your documentation as it appears the timezone isn't included when I've tested it?

**Comment by elroyairmed** [comment]
Are there problems with the system currently? Your website says the status is OK. The second flight (EK56) I've randomly tested isn't showing up at all and the times for the third flight (EK55) are out by 12 hours?

**Comment by troystaylor** [comment]
I can only provide limited support for the connector, not the service. I am not connected with or part of the backend service.
Have you tried using your API key in Postman? Have you tried authenticating with the connector a second time to see if the API key was entered incorrectly the first time?
